services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -7094,6 +7119,11 @@ public void configureActiveActiveReplication(
   public ArrayList<StoreInfo> getClusterStores(String clusterName) {
     // Return all stores at this step in the process
     return (ArrayList<StoreInfo>) getAllStores(clusterName).stream()
+        /**
+         * This filtering should not be needed because {@link #getAllStores(String)} should not return any null items,
+         * but just in case, we have it here as defensive code...
+         */
+        .filter(storeInfo -> storeInfo != null)
```

Comment:
We can also use `filter(Objects::nonNull)`

Reply:
Oh yeah nice, I thought there was something like that but I was blanking out...